src/components/ArticleCard/ArticleCard.css

```diff
@@ -0,0 +1,210 @@
+p {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+a {
+  color: inherit;
+}
```

Comment:
BEM doesn't allow using `tags` for styling in `css`. Please, add **class names**